apis/management.cattle.io/v3/cluster_types.go

```diff
@@ -103,6 +103,7 @@ type ClusterSpecBase struct {
 	EnableClusterMonitoring              bool                           `json:"enableClusterMonitoring" norman:"default=false"`
 	WindowsPreferedCluster               bool                           `json:"windowsPreferedCluster" norman:"noupdate"`
 	LocalClusterAuthEndpoint             LocalClusterAuthEndpoint       `json:"localClusterAuthEndpoint,omitempty"`
+	ScheduledClusterScan                 *ScheduledClusterScan          `json:"scheduledClusterScan,omitempty"`
```

Comment:
Maybe "ScanSchedule" makes sense instead of "ScheduledClusterScan"? It will be cluster.scanschedule?

Reply:
If we introduce some other kind of scan in future, then we won't be able to able to differentiate. ClusterScan is pretty specific.